.eslintrc.cjs

```diff
@@ -109,7 +109,7 @@ module.exports = {
 		"operator-assignment": "error",
 		"operator-linebreak": "error",
 		"max-len": ["error", { code: 100, ignoreTemplateLiterals: true }],
-		"max-params": ["error", 3],
+		"max-params": ["error", 4],
```

Comment:
We should leave eslint config as it is unless we discuss it first as a team. This should be the general style guide for all our code and we should not change it for some exceptions (e.g. express error middleware that takes 4 args). In those cases, it's better to just use `eslint-disable-next-line` to loosen up the project configuration.

Reply:
Got it.